app/src/sharedTest/java/org/oppia/android/app/topic/conceptcard/ConceptCardFragmentTest.kt

```diff
@@ -371,24 +371,24 @@ class ConceptCardFragmentTest {
   }
 
   @Test
-  fun testConceptCardFragment_launchSeveralConceptCardsWithSameSkill_onlyTheFirstOne() {
+  fun testConceptCardFragment_launchSeveralConceptCardsWithSameSkill_onlyTheFirstShows() {
     launchTestActivity().use { scenario ->
       scenario.onActivity { activity ->
         val fragment = ConceptCardFragment.bringToFrontOrCreateIfNew(
           TEST_SKILL_ID_0,
           ProfileId.getDefaultInstance(),
           activity.supportFragmentManager
         )
-        Assert.assertNotNull(fragment)
-        Assert.assertNull(
+        assertThat(fragment).isNotNull()
+        assertThat(
           ConceptCardFragment.bringToFrontOrCreateIfNew(
             TEST_SKILL_ID_0,
             ProfileId.getDefaultInstance(),
             activity.supportFragmentManager
           )
-        )
-        Assert.assertEquals(1, activity.supportFragmentManager.fragments.size)
-        Assert.assertEquals(fragment, activity.supportFragmentManager.fragments[0])
+        ).isNull()
+        assertThat(activity.supportFragmentManager.fragments.size).isEqualTo(1)
```

Comment:
```suggestion
        assertThat(activity.supportFragmentManager.fragments).hasSize(1)
```

More Truth-idiomatic version. Ditto elsewhere.

Reply:
Done